### PR TITLE
Marketplace user attributes

### PR DESCRIPTION
**Pull request by whscullin**
And latest EmbedSDK.

### CODE REVIEW

File: src/components/ApiFunctions/ApiFunctions.tsx [hunk 1]
```diff
@@ -130,6 +130,76 @@ export const ApiFunctions: React.FC<ApiFunctionsProps> = () => {
     history.push(`${ROUTES.CORESDK_ROUTE}?test=abcd`, { count: 1 })
   }
 
+  const getUserAttributeClick = () => {
+    extensionSDK
```
Comment (bryans99) [inline src/components/ApiFunctions/ApiFunctions.tsx:134]: Use async/await? Everywhere else does now (at least in this file).

File: src/components/ApiFunctions/ApiFunctions.tsx [hunk 2]
```diff
@@ -130,6 +130,76 @@ export const ApiFunctions: React.FC<ApiFunctionsProps> = () => {
...
+      .userAttributeGetItem('user_value')
+      .then((value) => {
+        updateMessages(`User attribute 'user_value' is ${value}`)
+      })
+      .catch(error => {
+        const name = `${extensionSDK.lookerHostData?.extensionId.replace('::', '_')}_user_value`
+        updateMessages(`Create a user attribute named "${name}" to use this attribute`)
+        console.error(error)
+      })
+    extensionSDK
```
Comment (bryans99) [inline src/components/ApiFunctions/ApiFunctions.tsx:144]: Add comment that this will fail?

**Review by bryans99 — CHANGES_REQUESTED** [review]
Couple of minor things. If you want to leave as is I will approve

File: src/components/ApiFunctions/ApiFunctions.tsx (hunk 1 above)
Comment (whscullin) [reply src/components/ApiFunctions/ApiFunctions.tsx:134]: Oh wow, I started this that long ago.

File: src/components/ApiFunctions/ApiFunctions.tsx (hunk 2 above)
Comment (whscullin) [reply src/components/ApiFunctions/ApiFunctions.tsx:144]: 👍

**Review by bryans99 — APPROVED** [review]
👍

**Comment by whscullin** [comment]
Marketplace user attributes and latest embed SDK

**Pull request merged**